src/components/AppLayout/index.tsx

```diff
@@ -134,6 +136,7 @@ const Layout: React.FC<Props> = ({
 }): React.ReactElement => {
   const [mobileNotSupportedClosed, setMobileNotSupportedClosed] = useState(false)
   const [expanded, setExpanded] = useState(false)
+  const [showMasterCopyError, setShowMasterCopyError] = useState(true)
```

Comment:
If you return `null` from `InvalidMasterCopyError` when it's valid then you no longer need this state.

Reply:
This is used to hide the error if the user closes the alert.
I moved this state into the InvalidMasterCopyError component as it is not needed outside of it.